### DISCUSSION TITLE
Set min. password length?

### DISCUSSION THREAD

**Discussion by sjau** [dani-garcia/vaultwarden#2326] [Ideas]
Is there a way to set the minimum password length for new users?

I noticed that new users have by default a password generated with ~16 characters. That's already better than probably what most people use without password manager, however I'd like to increase that to 50 chars.

Once you go into tools -> pw generator and set the length there, it will keep that. However new users have only like 16 chars set by default.

**Comment by jjlin** [comment]
If all your users belong to an organization, you could use the [Password Generator](https://bitwarden.com/help/policies/#password-generator) policy. Otherwise, you could build your own version of the web vault with the default changed, but that would only apply to the web vault, and not the other clients (browser, mobile, etc.).

BTW, 50 chars is objectively overkill. That is stronger than the 256-bit encryption key used to protect a vault. Anything beyond around 20 chars has pretty much no practical benefit.

**Reply by sjau** [reply]
> If all your users belong to an organization, you could use the [Password Generator](https://bitwarden.com/help/policies/#password-generator) policy.

It's about new users. 

> BTW, 50 chars is objectively overkill. That is stronger than the 256-bit encryption key used to protect a vault. Anything beyond around 20 chars has pretty much no practical benefit.

You say it's overkill (now) but does it hurt to have 50+ chars?
